.gitignore

```diff
@@ -169,4 +169,4 @@ tests/mxnet_unit_tests
 
 # generated wrappers for ccache
 cc
-cxx
\ No newline at end of file
+cxx
```

Comment:
if you moved out the wrappers then this few lines are not needed

Reply:
Will do